common/djangoapps/third_party_auth/docs/how_tos/migrating_apple_users_in_teams.rst

```diff
@@ -0,0 +1,43 @@
+Migrating Apple users while switching teams on Apple
+-----------------------------------------------
+
+This document explains how to migrate apple signed-in users in the event of
+switching teams on the Apple Developer console. When a user uses Apple to sign in,
+edx receives an `id_token from apple containing user information`_, including
```

Comment:
I would not refer to `edx` specifically, especially if we want this to be general Open edX documentation. Maybe say `LMS` or `edx-platform` instead.

Reply:
Thanks @dianakhuang. I replaced occurences of `edx` with `LMS`.